src/web/mjs/connectors/KomikTap.mjs

```diff
@@ -7,11 +7,11 @@ export default class KomikTap extends WordPressMangastream {
         super.id = 'komiktap';
         super.label = 'KomikTap';
         this.tags = [ 'manga', 'webtoon', 'hentai', 'indonesian' ];
-        this.url = 'https://komiktap.in';
+        this.url = 'https://92.87.6.124';
```

Comment:
IP shouldn't be hardcoded, just initialize `this.url` in the contructor by checking the http redirect
Same for canHandleURI, should be able to dynamically include the url pattern with the IP found
The request class has some redirect features, not sure if it can help